qa-test-apps/bundle-hash-test/README.md

```diff
@@ -0,0 +1,26 @@
+### In this example, we are checking bundle hash mismatch.
```

Comment:
Sorry. I am not sure I understand why / what to we need to ignored here (probably because I am missing context over the issue) Can you please explain?